spec/project_configuration_spec.rb

```diff
@@ -128,6 +128,38 @@
         expect(config.path).to eq 'Another Path'
       end
     end
+    
+    describe '#schemes' do
+      it 'returns an array of schemes' do
```

Comment:
Since this test is only testing that the `attr_accessor` works as expected I don't think it's necessary since we can rely on ruby to handle that properly. Thoughts?